src/Utils/Json.php

```diff
@@ -54,4 +54,23 @@ public static function decode(string $json, int $flags = 0)
 		}
 		return $value;
 	}
+
+	/**
+	 * Decodes a JSON string to a stdClass
+	 * @return \stdClass
+	 */
+	public static function decodeToObject(string $json)
+	{
+		return self::decode($json);
```

Comment:
Imho `["foo","bar"]` would fail here. Typecasting is needed.

Reply:
would fail where? $json is type hinted as a string as it is in the main decode method.

Reply:
Output of json_decode depends on json given in input. You could enforce array with FORCE_ARRAY, but there is no option FORCE_OBJECT

Reply:
json_decode returns stdClass with no second parameter given (or FALSE) or an associative array with second parameter set to true. Or am I still missing something?

Reply:
The behaviour of Json::decode is the same (just since 3.0 the boolean second parameter changed to integer represented with Json::FORCE_ARRAY)

Reply:
http://sandbox.onlinephpfunctions.com/code/e22ec5b9f52f2f6f7fb4104efcad5ba3247a9e48

Reply:
Oh, I get it now. Non-associative arrays.